templates/templates/_navigation.html

```diff
@@ -1,8 +1,8 @@
 {% set breadcrumbs = get_navigation(request.path).breadcrumbs %}
 <header id="navigation" class="p-navigation--sliding is-dark {% if breadcrumbs and breadcrumbs.children or breadcrumbs.grandchildren %}is-reduced {% elif breadcrumbs == 'tutorials' %}is-reduced{% endif %}">
   {% block header_banner %}
-  <div class="row">
-    <div class="p-navigation__banner col-3">
+  <div class="p-navigation__row--25-75">
+    <div class="p-navigation__banner col">
```

Comment:
`.col` is not needed, as the navigation always consists of the __banner and __nav, they are targeted directly.

```suggestion
    <div class="p-navigation__banner">
```